boxsdk/client/client.py

```diff
@@ -317,6 +318,39 @@ def create_group(self, name):
         )
 
     @api_call
+    def get_recent_items(self, limit=None, marker=None, fields=None, **kwargs):
```

Comment:
In docstring, document that `**kwargs` are passed to `MarkerBasedObjectCollection`.

We could also call this `**collection_kwargs` or `**paging_kwargs`.